content/chat/setup.textile

```diff
@@ -0,0 +1,102 @@
+---
+title: SDK setup
+meta_description: "Install, authenticate and instantiate the Chat SDK."
+product: chat
+languages:
+  - javascript
+---
+
+Use these instructions to install, authenticate and instantiate the Chat SDK.
+
+h2(#authentication). Authentication
```

Comment:
I think we should have two sections on auth, or somehow clarify it's two rather distinct things that need to be done.

One is the Ably API key stuff (authenticating yourself as an app/dev with Ably), and permissions for the API KEY itself.

Another one is authenticating your users. Here you have the option to use the api key directly but you can also use tokens/etc.

Reply:
The content is here, might just need an extra title or something?

I think @AndyTWF mentioned this in standup at some point - maybe a page for auth that's separated from the rest of the setup? The setup can include "Sign up to Ably and get an API key" but doesn't need to go into anything else.

Reply:
I think if we're going to do this then it should be in a separate PR after we've merged these, personally.